code/modules/defenses/sentry.dm

```diff
@@ -521,10 +521,22 @@ obj/structure/machinery/defenses/sentry/premade/damaged_action()
 	desc = "A deployable, omni-directional automated turret with AI targeting capabilities. Armed with an M30 Autocannon and a 1500-round drum magazine.  Due to the deployment method it is incapable of being moved."
 	ammo = new /obj/item/ammo_magazine/sentry/dropped
 	faction_group = FACTION_LIST_MARINE
-	luminosity = 5
 	omni_directional = TRUE
 	immobile = TRUE
 	static = TRUE
+	var/obj/structure/machinery/camera/cas/linked_cam
+	var/static/sentry_count = 1
+	
+/obj/structure/machinery/defenses/sentry/launchable/Initialize()
+	. = ..()
+	linked_cam = new(loc, "[name] [sentry_count]")
+	SetLuminosity(7)
```

Comment:
```suggestion
	SetLuminosity(5)
```
should it not be?

Reply:
The base sentry code has the luminosity set to 7 when you turn it on and off so this is easier (and standardized). If you really want me to I'll go make a variable for luminosity setting and change it for this.